src/modules/partition/PartitionViewStep.cpp

```diff
@@ -616,7 +616,12 @@ PartitionViewStep::onLeave()
 
             // If the root partition is encrypted, and there's a separate boot
             // partition which is not encrypted
-            if ( root_p->fileSystem().type() == FileSystem::Luks && boot_p->fileSystem().type() != FileSystem::Luks )
+            if ( ( root_p->fileSystem().type() == FileSystem::Luks && boot_p->fileSystem().type() != FileSystem::Luks )
+#ifdef WITH_KPMCORE4API
```

Comment:
Note that in Calamares 3.3 (which you're merging to), KPMCore4 is a given, and there is no `#define` for it anymore.

Reply:
Good to know thanks. I'll remove the `#ifdefs`

Reply:
Done